src/gax-internal/src/api_header.rs

```diff
@@ -48,7 +48,25 @@ impl XGoogApiClient {
         let gax_version = build_info::PKG_VERSION;
```

Comment:
Observation: gax version is really gaxi version. gaxi is more interesting to us

Reply:
ack. go/cloud-api-headers says it should be called `gax/`.  We could try to capture both.